core/base/src/TApplication.cxx

```diff
@@ -1437,12 +1437,12 @@ Longptr_t TApplication::ProcessLine(const char *line, Bool_t sync, Int_t *err)
    }
 
    if (!strcmp(line, ".reset")) {
-      // Do nothing, .reset disabled in CINT because too many side effects
+      // Do nothing, .reset disabled in Cling because too many side effects
       Printf("*** .reset not allowed, please use gROOT->Reset() ***");
       return 0;
 
 #if 0
-      // delete the ROOT dictionary since CINT will destroy all objects
+      // delete the ROOT dictionary since Cling will destroy all objects
```

Comment:
This code is disabled (see the `#if 0`) so maybe it was really referring to CINT.

Reply:
Indeed, good catch. In fact, that disabled part of the code hasn't been modified in 22y. Maybe it could be even removed. But I just reverted it.